k8s/aat/common/rpe/bulk-scan-payment-processor.yaml

```
@@ -0,0 +1,41 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: bulk-scan-payment-processor
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: bulk-scan-payment-processor
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: bulk-scan-payment-processor
+    version: 1.0.0
```

```suggestion
    version: 0.0.2
```
this is the version of chart - https://github.com/hmcts/bulk-scan-payment-processor/blob/master/charts/bulk-scan-payment-processor/Chart.yaml#L6